protocol/x/affiliates/types/keys.go

```diff
@@ -16,4 +16,6 @@ const (
 	ReferredVolumeKeyPrefix = "ReferredVolume:"
 
 	AffiliateTiersKey = "AffiliateTiers"
+
+	AffiliateWhitelistKey = "AffiliateWhitelist"
```

Comment:
Let's actually optimize the keys and prefixes a bit (for faster insert/read from IAVL tree)
```
	ReferredByKeyPrefix = "RB:"
	ReferredVolumeKeyPrefix = "RV:"

	AffiliateTiersKey = "AT"

	AffiliateWhitelistKey = "AW"
	```